taichi/ir/statements.h

```diff
@@ -248,6 +248,28 @@ class LoopIndexStmt : public Stmt {
   TI_DEFINE_ACCEPT_AND_CLONE
 };
 
+// All loop indices fused together
+class LoopLinearIndexStmt : public Stmt {

```

Comment:
What does this statement return if `index` is unused? Something like a `LinearizeStmt`?

Reply:
Yeah, basically linearize all the indices. This maps to `threadIdx` in CUDA :-)